### ISSUE TITLE
Can't Recognize the Commands in xf 4.1 vs 2019

### ISSUE THREAD

**Issue by BillyMartin1964** [nuitsjp/Xamarin.Forms.GoogleMaps.Bindings#22]
Keep getting errors on all of the command bindings in the xaml page.
Error	XLS0413	The property 'Command' was not found in type 'MapClickedToCommandBehavior'.

    xmlns:bindings="clr-namespace:Xamarin.Forms.GoogleMaps.Bindings;assembly=Xamarin.Forms.GoogleMaps.Bindings"

```
 <googleMaps:Map VerticalOptions="FillAndExpand">
            <googleMaps:Map.Behaviors>
                <bindings:MapClickedToCommandBehavior Command="{Binding MapClickedCommand}" />
                <bindings:BindingPinsBehavior Value="{Binding WindPins}" />
                <bindings:PinClickedToCommandBehavior Command="{Binding PinClickedCommand}" />
                <bindings:SelectedPinChangedToCommandBehavior Command="{Binding SelectedPinChangedCommand}" />
                <bindings:InfoWindowClickedToCommandBehavior Command="{Binding InfoWindowClickedCommand}" />
                <bindings:InfoWindowLongClickedToCommandBehavior Command="{Binding InfoWindowLongClickedCommand}" />
                <bindings:PinDragStartToCommandBehavior Command="{Binding PinDragStartCommand}" />
                <bindings:PinDraggingToCommandBehavior Command="{Binding PinDraggingCommand}" />
                <bindings:PinDragEndToCommandBehavior Command="{Binding PinDragEndCommand}" />
            </googleMaps:Map.Behaviors>
        </googleMaps:Map>

```

![image](https://user-images.githubusercontent.com/29313458/63187727-95c09b80-c02d-11e9-8469-1d59862966e8.png)

**Comment by mharrisonbit** [comment]
I am having the same issue, I was hoping that there would be a solution for this.

**Comment by QuangKhanDo** [comment]
yeah, me either.

**Comment by orwo1** [comment]
Is there going to be an update to resolve this issue?

**Comment by valentasm1** [comment]
This PR solves it. Could anyone merge it and release nuget?
https://github.com/nuitsjp/Xamarin.Forms.GoogleMaps.Bindings/pull/21

**Comment by nuitsjp** [comment]
I'm sorry it took so long.
I've just released Version 3.3.0 after taking in #21  .

- https://github.com/nuitsjp/Xamarin.Forms.GoogleMaps.Bindings/releases/tag/Release-3.0.0

Please use this version from now on.
Thank you for your cooperation.
